tests/integration/test_types_integration.py

```diff
@@ -280,12 +280,12 @@ def test_timestamp(trino_connection):
 
 
 def test_timestamp_with_timezone(trino_connection):
```

Comment:
Squash "Refactor some test function names for consistency" since it changes something introduced in the first commit itself.